reco_utils/dataset/wikidata.py

```diff
@@ -44,7 +44,8 @@ def find_wikidata_id(name, limit=1, session=None):
         session (requests.Session): requests session to reuse connections
 
     Returns:
-        str: Wikidata entityID corresponding to the title string. 'entityNotFound' will be returned if no page is found.
+        (str): wikidata entityID corresponding to the title string.
```

Comment:
We follow Google style, which does not use parentheses for return types. 
I.e. you should have 
`str: wikidata entityID corresponding to the title string.`
not 
`(str): wikidata entityID corresponding to the title string.`
See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html